azerothcore/azerothcore-wotlk
# fix(Scripts/Paladin) Infusion of Light proc.

## Changes Proposed:
-  Fix infusion of light talent of paladin with sacred shield on does not proc.
-  I only make a PR with change propossed by @PsYchoNauTek in this issue https://github.com/azerothcore/azerothcore-wotlk/issues/11930 

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/11930

## SOURCE:


## Tests Performed:

- Build on Windows 10 and with Visual Studio 2022 
- Run authserver.exe and worldserver.exe OK!
- Test in-game all spells related to the fix and all work propertly.


## How to Test the Changes:


1. Create a Paladin (Levelup to 80) and learn spells.
2. Set talent Infusion of light (53576).
3. Cast Sacred shock (48825) until critic hit.
4. Then cast Sacred Shield (53601).
5. And until Sacred Shield is active cast Beam of light (48785).
6. Then the HoT Beam of light (66922) should be activated.

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
